### ISSUE TITLE
Refactor Yarn Packages

### ISSUE THREAD

**Issue by pahund** [zapperment/zapperment#81]
* Refactor normalization and neural network training code into extra Yarn package
* Refactor storage module into extra Yarn package
